src/__test__/Home.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { render, screen } from '@testing-library/react';
+import Home from '../components/Home';
+import '@testing-library/jest-dom';
+
```

Comment:
**[Optional]**

- [ ] This is a pair-programming project but I don't see any commit from your partner side :thinking: Kindly ask your partner to contribute to this project to follow the project requirement and also to make your work professional :+1: